presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -331,3 +331,9 @@ can use the following rules:
 
 .. literalinclude:: system-information-access.json
     :language: json
+
+A fixed user can be set for management interfaces using the ``management.user``
```

Comment:
It is not clear if this management user only applies to the system information or is a general thing. If it is a general thing (and I think it is) .. this needs to be a separate section with its own title and then linked from the current section. Otherwise this looks good.

Reply:
It only applies to system information